### ISSUE TITLE
pythonPackages.pendulum fails with missing setup.py

### ISSUE THREAD

**Issue by mweinelt** [NixOS/nixpkgs#95204]
**Describe the bug**
```
Executing setuptoolsBuildPhase
Traceback (most recent call last):
  File "nix_run_setup", line 8, in <module>
    exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
IOError: [Errno 2] No such file or directory: 'setup.py'
builder for '/nix/store/4jsxdfl8xa3ys7cnxraixk8b0jg1bh0r-python2.7-pendulum-2.1.2.drv' failed with exit code 1
```

**To Reproduce**
1. git checkout master
2. nix-build -A python3Packages.pendulum

**Notify maintainers**

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: python3Packages.pendulum
# a list of nixos modules affected by the problem
module:
```

**Comment by FRidh** [comment]
fixed in d58b4c9a14b97312defa629187cc8791c8d73c46

**Comment by mweinelt** [comment]
Hah, tried the same with poetry itself, and wondered where I went wrong.
